cypress/support/pages/calendar/pageCalendar.js

```diff
@@ -2,11 +2,80 @@
 
 class Calendar {
 	static #calendarOverviewNavigationButton = '[data-testid="Termine"]';
+	static #calendarOverviewPageTitle = '[data-testid="Kalender"]';
+	static #calendar = '[id="calendar"]';
+	static #nextCalendarPage = '[aria-label="next"]';
+	static #eventNameFormElement = '[data-testid="team_event_name"]';
+	static #eventStartDateFormElement = '[data-testid="form-datetime-input-startDate"]';
+	static #eventEndDateFormElement = '[data-testid="form-datetime-input-endDate"]';
+	static #eventDescriptionFormElement = '[data-testid="team_event_description"]';
+	static #eventLocationFormElement = '[data-testid="team_event_location"]';
+	static #eventFormSubmitButton = '[data-testid="submit-btn-create-event-modal"]';
+	static #eventFormDeleteButton = '[data-testid="calendar-event-btn-delete"]';
 
 	navigateToCalendarOverview() {
 		cy.visit("/calendar");
 		cy.get(Calendar.#calendarOverviewNavigationButton).click();
 		cy.url().should("include", "/calendar");
 	}
+
+	seeCalendarOverviewPage() {
+		cy.url().should("include", "/calendar");
+		cy.get(Calendar.#calendarOverviewPageTitle).should("exist");
+	}
+
+	clickOnDay(daynumber) {
+		cy.get(Calendar.#calendar).find("span").contains(daynumber).click();
+	}
+
+	clickOnNextCalendarPage() {
+		cy.get(Calendar.#nextCalendarPage).eq(1).click();
+	}
+
+	seeCreateEventModal() {
+		cy.get(Calendar.#eventNameFormElement).should("be.visible");
+	}
+
+	enterEventTitle(eventTitle) {
+		cy.get(Calendar.#eventNameFormElement).eq(1).type(eventTitle);
+	}
+
+	enterEventStartTime(eventStartTime) {
+		cy.get(Calendar.#eventStartDateFormElement).eq(1).type("{backspace}".repeat(4)).type(eventStartTime);
```

Comment:
I agree that there are two identical selector but if you see in the parent structure there one have hidden type and another doesn't have it. For the intended selector you don't have this in parent structure.

`<input type="hidden" name="_method" data-force-value="true" value="put">`

it means that the first form includes a `hidden` input with the name `_method` and a value of `put`. This suggests that the first form is intended for updating an existing event, while the second form is likely used for creating a new event. Its much better to check for these attributes rather hardcoding them. 

if you can deal it in your upcoming ticket then its fine for now but keep these points in the AC of that ticket

Reply:
will be resolved in a later ticket